domains/shafi.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "A space for Shafi. Shafi is working as a web developer for 4+ years.",
+  "repo": "https://github.com/shafi-/shafi-.github.io",
+  "owner": {
+    "username": "shafi-",
+    "email": "shafi.cse.buet@gmail.com"
+  },
+  "record": {
+    "CNAME": "shafi-.github.io"
```

Comment:
@shafi-  This cname record is invalid. I'm having trouble publishing it because of the hyphen at the end. You will have to use a different static deployment service with custom domain support.